File: C1.UWP.Calendar/VB/CalendarData/Data/AppointmentCollection.vb
Imports System.Threading.Tasks
Imports System.Text
Imports System.Linq
Imports System.Collections.ObjectModel
Imports System.Collections.Generic
Imports System

' inherit from ObservableCollection to allow auto refresh in calendar
Public Class AppointmentCollection
		Inherits ObservableCollection(Of Appointment)
		Public Sub New()
		End Sub
	End Class

File: C1.UWP.Calendar/CS/CalendarData/Data/AppointmentCollection.cs
using System;
using System.Collections.Generic;
using System.Collections.ObjectModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CalendarData
{
    // inherit from ObservableCollection to allow auto refresh in calendar
    public class AppointmentCollection : ObservableCollection<Appointment>
    {
        public AppointmentCollection()
        {
        }
    }
}
